engine/client/in_sdl.c

```diff
@@ -1099,7 +1099,7 @@ void Sys_SendKeyEvents(void)
 	}
 	else if (Key_Dest_Has(kdm_game))
 		osk |= sys_osk.ival;
-	if (osk)
+	if (false)
```

Comment:
Nit pick: Could you please wrap this around an `#if 0` instead and add a comment with an explanation as to why this is necessary? Thanks!